acceptance/Rakefile

```diff
@@ -18,16 +18,11 @@ module HarnessOptions
     :type => 'aio',
     :helper => ['lib/helper.rb'],
     :tests  => ['tests'],
-    :log_level => 'info',
```

Comment:
Thanks for cleaning this up; I didn't know enough about beaker to be sure which options were useless